src/pages/events/createEvents/CreateEventCriteria.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import useCreateEventStyles from '../../../assets/styleJs/events/createEvent'
+import {Box, Button, Typography} from '@mui/material'
+import CriteriasPapers from './CriteriasPaper'
+
+const CreateEventCriteria = () => {
+  const classes = useCreateEventStyles()
+
+  return (
+    <Box>
+      <Box className={classes.infoRootBoxCriteria}>
+        <Box className={classes.criteriaHeaderBox}>
+          <Typography className={classes.criteriaHeaderText}>
+            {' '}
+            Choose Evaluation Criteria{' '}
```

Comment:
What is point of this, use padding if needed

Reply:
done